### ISSUE TITLE
Add Class function

### ISSUE THREAD

**Issue by Joshcarp** [anz-bank/decimal#41]
This should return a string with a sign and the type of the decimal number 
eg `Class(0)  --> +0`
`Class(-420) --> +Normal`

Also consider adding a zero flavour in the constants `flZero`

**Comment by Joshcarp** [comment]
from specification:
```class(x) tells which of the following ten classes x falls into:
signalingNaN
quietNaN
negativeInfinity
negativeNormal
negativeSubnormal
negativeZero
positiveZero
positiveSubnormal
positiveNormal
positiveInfinity.```

**Comment by Joshcarp** [comment]
fixed in #47
